airflow/providers/common/sql/operators/sql.py

```diff
@@ -221,6 +220,16 @@ def execute(self, context):
 
         return output
 
+    def on_kill(self) -> None:
+        self.log.info("Stopping:", self._hook.query_ids)
```

Comment:
here we don't know if we will stop the query as we don't know if the hook implemented the needed function